src/components/AddItemForm.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { collection, addDoc } from 'firebase/firestore';
+import { db } from '../lib/firebase';
+import './AddItemForm.css';
+
+function AddItemForm() {
+  const submitItem = (event) => {
+    event.preventDefault();
+    const token = window.localStorage.getItem('userToken');
```

Comment:
Wonder if you could call this const userToken so you don't need to do the reassignment on line 13.